src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -87,9 +89,11 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
 - (BOOL)pairDevice:(uint64_t)deviceID
      discriminator:(uint16_t)discriminator
       setupPINCode:(uint32_t)setupPINCode
-             error:(NSError * __autoreleasing *)error;
+             error:(NSError * __autoreleasing *)error DEPRECATED_MSG_ATTRIBUTE("Use setupCommissioningSessionWithPayload method.");
```

Comment:
Please don't.  We'll add deprecation messages here soon, and this will just cause merge conflicts with other ongoing work....  And this is not the right deprecation macro in any case.

Reply:
OK with this as long as there's a plan to add these deprecation messages, otherwise its not clear that this isn't an API that should be used, will revert